### ISSUE TITLE
Provide support for other databases

### ISSUE THREAD

**Issue by andreasscherbaum** [Icinga/icinga2-ansible#47]
The Ansible Playbook only supports MySQL, can other databases (like PostgreSQL) be supported as well?

**Comment by ctaperts** [comment]
I would look at adding in another role like bas-ansible-roles-collection.postgresql9-server

psql is not a straight forward setup

**Comment by andreasscherbaum** [comment]
Right now this role is "icinga2-ansible", and judged by the name it should cover all of them. Otherwise this role should be renamed as well.

It's straightforward to move Playbooks for every database into a separate file, and include the files from the main.yml, based on the selected database. It can even default to "MySQL", to make transition easier.

**Comment by aflatto** [comment]
Yes, we would love to incorporate the Postgresql Support for the Ansible playbook, do you have a Pull request you wish to submit ?

**Comment by rubdos** [comment]
> I am not a dev on this project but I would look at adding in another role like bas-ansible-roles-collection.postgresql9-server

Why? I would leave the setup of PostgreSQL to be done by the playbook developer. That way, he can pick the role and setup he likes. Maybe the host that'll host Icinga web2 already has PostgreSQL installed (which is my case), and we only need to create a role and database...
